metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/files/bro_index.template

```diff
@@ -98,7 +98,7 @@
           "mapping": {
             "type": "float"
           },
-          "match": "threat.triage.rules:*:score",
+          "match": "threat:triage:*score",
```

Comment:
Prior to this change, the overall threat triage score field `threat:triage:score` was not mapped to a `float` as we had assumed.

Reply:
great catch.